src/checkboxes/template.test.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Is it possible to do this the other way around and opt into puppeteer since jsdom seems like a better default for us?

Reply:
done